syntaxes/language-configuration.json

```diff
@@ -0,0 +1,14 @@
+{
+  "wordPattern": "[a-zA-Z0-9_-]+",
+  "brackets": [
+    ["{", "}"],
+    ["[", "]"],
+    ["(", ")"]
+  ],
+  "onEnterRules": [
```

Comment:
I'm not sure about that, as it applies to fixed format source code only. Is there a way to make it configurable in some way or another?
(Do you know how general is `action`? — can it be a custom command?)

Reply:
Or is it indeed conditional on the line starting with 6 digits?

Reply:
After reading the doc: (too bad `appendText` seems to be fixed…). Anyways, just from the description this seems to be legit and general enough (but I did not test).